css/styles.css

```diff
@@ -64,7 +66,7 @@ a:visited {
 
 a:hover {
   font-weight: bold;
-  text-shadow: 0 0 0.77px grey;
+  text-shadow: 0 0 2.77px white;
   cursor: pointer;
 }
 
```

Comment:
abschließend nur noch als Kommentar: Super Seite! 
Teile das CSS gerne noch in einzelne Teile auf :) 
ansonsten tippi toppi 🏅